apps/els_lsp/src/els_providers_sup.erl

```diff
@@ -45,8 +45,8 @@ start_link() ->
 -spec init([]) -> {ok, {supervisor:sup_flags(), [supervisor:child_spec()]}}.
 init([]) ->
   SupFlags = #{ strategy  => one_for_one
-              , intensity => 1
-              , period    => 5
+              , intensity => 5
```

Comment:
Relaxed a bit to avoid the whole server to stop if a provider is occasionally misbehaving.